.circleci/config.yml

```diff
@@ -140,9 +144,11 @@ jobs:
       - store_artifacts:
           path: coverage.xml
 
+  # Try to immitate commands executed by the readthedocs build
+  # https://readthedocs.org/projects/pytorch-lightning/builds/
   build-Docs:
     docker:
-      - image: readthedocs/build:latest
+      - image: readthedocs/build:ubuntu-20.04-2022.02.16
```

Comment:
`readthedocs/build:latest` is two years old (see [here](https://hub.docker.com/layers/build/readthedocs/build/latest/images/sha256-3bc8a04cafd51ca7520b1c0c4b002f6aaac52695fa4d9f69c3ee92fac689678c?context=explore)), and I'm quite sure that it isn't used by readthedocs.

Reply:
very nice finding! :rabbit: